modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/components/SubscriptionPlanRow.kt

```diff
@@ -138,6 +155,7 @@ private fun SubscriptionPlanRow(
                 .fillMaxWidth()
                 .clip(RoundedCornerShape(8.dp))
                 .background(MaterialTheme.theme.colors.primaryUi01Active)
+                .padding(vertical = rowConfig.verticalPadding)
```

Comment:
This padding should be applied after clickable. Otherwise ripple won't cover the whole component.